adam_preprocessing/utils.py

```diff
@@ -231,6 +231,52 @@ def accuracy(output, target, topk=(1,)):
     return res
 
 
+def get_situation_accuracy(
+    output: torch.Tensor,
+    target: torch.Tensor,
+    situation_number_to_object_indices: Mapping[int, Sequence[int]],
+) -> float:
+    """
+    Computes the situation-wise acurracy@k for the specified values of k.
```

Comment:
This docstring makes it sound like the function returns the accuracy of a particular situation, i.e. `situation_k`. I would not mention the variable `k` in this docstring and instead just say that the function calculates situation-wise accuracy across all situations

Reply:
My bad, that came from the original accuracy function's docstring. I'll remove it.

Reply:
Removed the `@k` part from the docstring.